js/script.js

```diff
@@ -1 +1,88 @@
-console.log('Hello, front end');
+var input = document.getElementById('form');
+var radioF = input.tempInF;
+var radioC = input.tempInC;
+radioF.addEventListener('click', radioSelector(radioF));
+radioC.addEventListener('click', radioSelector(radioC));
+function valueTest () {
+    if ((parseInt(input.tempToConvert.value)) || input.tempToConvert.value === "0") {
+        return true;
+    } else {
+      document.getElementsByClassName('inputBox')[0].value = "Please enter a number";
+      return false;
+    }
+}
+
+function radioSelector(option) {
+  if (option.checked === false) {
+    option.checked = true;
+  }
+  return option;
+
+}
+function findComfortLevel() {
+    var comfort = "";
+    if (radioC.checked === true) {
+      if (convertTemp() > 89) {
+        comfort = "hot";
+      } else if (convertTemp() < 50) {
+        comfort = "cold";
+      } else {
+        comfort = "warm";
+      }
+    } else {
+      if (convertTemp() > 31) {
+          comfort = "hot";
+      } else if (convertTemp() < 10) {
+          comfort = "cold";
+      } else {
+          comfort = "warm";
+      }
+    }
+    return comfort;
+  }
+
+function round(value, decimals) {
+  return Number(Math.round(value + 'e' + decimals) + 'e-' + decimals);
+}
+
+function convertTemp() {
+  var currentValue = input.tempToConvert.value;
+  if (radioF.checked === true) {
+    // input.tempInF.checked = false;
+    currentValue = (currentValue - 32) / 1.8;
+
+  } else {
+    // input.tempInC.checked = false;
+    currentValue = (currentValue * 1.8) + 32;
+  }
+  return round(currentValue, 1);
+}
+
+function drawOutput() {
+  var isNumber = valueTest();
+  if (isNumber) {
```

Comment:
If you wanted to avoid another "var" you could also call the function directly in your "if" statement

`if( valueTest() )`